configurations/longevity-fips-and-encryptions.yaml

```diff
@@ -1,4 +1,5 @@
-ami_id_db_scylla: 'ubuntu-pro-fips-server/images/hvm-ssd/ubuntu-focal-20.04-amd64-pro-fips-server-20221121-7bc828d1-c072-4d33-a989-fbad50380cfb'
+ami_id_db_scylla: 'resolve:ssm:/aws/service/marketplace/prod-k6fgbnayirmrc/latest' # Ubuntu Pro FIPS 20.04 LTS amd64
+linux_distro_features: ['fips']
```

Comment:
I'd call it `assert_linux_distro_features`

Reply:
change the name of it